helm-charts/helm-repo/ibm_cloud_pak/manifest.yaml

```diff
@@ -0,0 +1,69 @@
+images:
+- image: ibm-helm-repo-operator:3.5.0
+  references:
+  - repository: ibm-helm-repo-operator:3.5.0
+    pull-repository: quay.io/opencloudio/ibm-helm-repo-operator:3.5.0
+    platform:
+      os: linux
+      arch: amd64
+  - repository: ibm-helm-repo-operator:3.5.0
+    pull-repository: quay.io/opencloudio/ibm-helm-repo-operator:3.5.0
+    platform:
+      os: linux
+      arch: ppc64le
+  - repository: ibm-helm-repo-operator:3.5.0
+    pull-repository: quay.io/opencloudio/ibm-helm-repo-operator:3.5.0
+    platform:
+      os: linux
+      arch: s390x
+- image: icp-helm-repo@sha256:bc95dc9703c0d26ad3af1004bbe4b236781ebd84082df430108948897dfabfe2
+  references:
+  - repository: icp-helm-repo@sha256:bc95dc9703c0d26ad3af1004bbe4b236781ebd84082df430108948897dfabfe2
+    pull-repository: quay.io/opencloudio/icp-helm-repo@sha256:bc95dc9703c0d26ad3af1004bbe4b236781ebd84082df430108948897dfabfe2
+    platform:
+      os: linux
+      arch: amd64
+  - repository: icp-helm-repo@sha256:bc95dc9703c0d26ad3af1004bbe4b236781ebd84082df430108948897dfabfe2
+    pull-repository: quay.io/opencloudio/icp-helm-repo@sha256:bc95dc9703c0d26ad3af1004bbe4b236781ebd84082df430108948897dfabfe2
+    platform:
+      os: linux
+      arch: ppc64le
+  - repository: icp-helm-repo@sha256:bc95dc9703c0d26ad3af1004bbe4b236781ebd84082df430108948897dfabfe2
+    pull-repository: quay.io/opencloudio/icp-helm-repo@sha256:bc95dc9703c0d26ad3af1004bbe4b236781ebd84082df430108948897dfabfe2
+    platform:
+      os: linux
+      arch: s390x
+- image: icp-audit-service:3.4.0
+  references:
+  - repository: icp-audit-service:3.4.0
+    pull-repository: quay.io/opencloudio/icp-audit-service:3.4.0
+    platform:
+      os: linux
+      arch: amd64
+  - repository: icp-audit-service:3.4.0
+    pull-repository: quay.io/opencloudio/icp-audit-service:3.4.0
+    platform:
+      os: linux
+      arch: ppc64le
+  - repository: icp-audit-service:3.4.0
+    pull-repository: quay.io/opencloudio/icp-audit-service:3.4.0
+    platform:
+      os: linux
+      arch: s390x
+- image: icp-audit-service@sha256:334fae7bd3ae4c04611f4e82207a2f7836dbc589f7fc2b3cd172bd93572bfe16
+  references:
+  - repository: icp-audit-service@sha256:334fae7bd3ae4c04611f4e82207a2f7836dbc589f7fc2b3cd172bd93572bfe16
+    pull-repository: quay.io/opencloudio/icp-audit-service@sha256:334fae7bd3ae4c04611f4e82207a2f7836dbc589f7fc2b3cd172bd93572bfe16
+    platform:
+      os: linux
+      arch: amd64
+  - repository: icp-audit-service@sha256:334fae7bd3ae4c04611f4e82207a2f7836dbc589f7fc2b3cd172bd93572bfe16
+    pull-repository: quay.io/opencloudio/icp-audit-service@sha256:334fae7bd3ae4c04611f4e82207a2f7836dbc589f7fc2b3cd172bd93572bfe16
+    platform:
+      os: linux
+      arch: ppc64le
+  - repository: icp-audit-service@sha256:334fae7bd3ae4c04611f4e82207a2f7836dbc589f7fc2b3cd172bd93572bfe16
+    pull-repository: quay.io/opencloudio/icp-audit-service@sha256:334fae7bd3ae4c04611f4e82207a2f7836dbc589f7fc2b3cd172bd93572bfe16
```

Comment:
icp-audit-service:3.4.1 digest is `24a220374c39652f8b031637d852995bfaec5be2601f9e4c41eb1925531f466d`